content/post/2023-06-06-system-dependencies/index.Rmd

```diff
@@ -0,0 +1,259 @@
+---
+slug: system-dependency
+title: "System Dependencies in R Packages & Automatic Testing" 
+authors: 
+- Hugo Gruson
+date: "2023-06-06" 
+tags: 
+- package development 
+- r-package
+output: hugodown::hugo_document
+---
+
+<!-- FIXME: Use this image for social media: https://pixabay.com/fr/photos/alice-anglais-pays-des-merveilles-2902560/ -->
+
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(fig.path = "", comment = "", warning = FALSE)
+# knitr hook to make images output use Hugo options
+knitr::knit_hooks$set(
+  plot = function(x, options) {
+    hugoopts <- options$hugoopts
+    paste0(
+      "{{<figure src=",
+      '"', x, '" ',
+      if (!is.null(hugoopts)) {
+        glue::glue_collapse(
+          glue::glue('{names(hugoopts)}="{hugoopts}"'),
+          sep = " "
+        )
+      },
+      ">}}\n"
+    )
+  }
+)
+options(crayon.enabled = FALSE)
+
+```
+
+In a previous post, we discussed about a package dependency that goes slightly beyond the normal R package ecosystem dependency: R itself.
+Today, we step even further and discuss about dependencies that have nothing to do with R: system dependencies.
```

Comment:
```suggestion
Today, we step even further and discuss dependencies that have nothing to do with R: system dependencies.
```

I'd rephrase "nothing to do with R", because we still want to use them from R.

Reply:
I'd recommend adding examples of system dependencies and related packages right in the intro, to clarify the topic.